src/storage/v2/disk/storage.cpp

```diff
@@ -1607,7 +1822,16 @@ void DiskStorage::DiskAccessor::Abort() {
 }
 
 void DiskStorage::DiskAccessor::FinalizeTransaction() {
+  /// TODO: (andi) Check the login in InMemoryStorage.
   if (commit_timestamp_) {
+    auto *disk_storage = static_cast<DiskStorage *>(storage_);
+    bool edge_import_mode_active = disk_storage->edge_import_status_ == EdgeImportMode::ACTIVE;
+
+    if (edge_import_mode_active) {
+      auto &committed_transactions = disk_storage->edge_import_mode_cache_->GetCommittedTransactions();
```

Comment:
Could you please tell me where do we read committedTxs? As I understood we don't use it now.

Reply:
nowhere but deltas are stored in tx, so I have to preserve transsactions

Reply:
Can we add a comment about this? So if somebody will want to remove "unnecessary" member will read a comment and keep the member :)